.husky/helpers/prevent_conflict_markers.sh

```diff
@@ -4,9 +4,12 @@ repeat() {
   char="$1"
   times="$2"
   text=""
-  for (( i = 0; i < "$times"; ++i ))
+  i="0"
```

Comment:
Are there any disadvantages of keeping this as an integer itself rather than a string?

Reply:
I have modified it to integer value.